--- a/lib/python2.7/site-packages/vmtk/vmtkrenderer.py	2018-03-21 02:13:02.000000000 +0000
+++ b/lib/python2.7/site-packages/vmtk/vmtkrenderer.py	2018-12-19 09:18:20.000000000 +0000
@@ -330,7 +330,7 @@
             #self.TextActorStd.SetPosition(self.PositionStd)
             #self.Renderer.AddActor(self.TextActorStd)
 
-            fontSize = self._GetScreenFontSize()
+            fontSize = int(self._GetScreenFontSize())
 
             self.TextActor = vtk.vtkTextActor()
             self.TextActor.GetTextProperty().SetFontSize(fontSize)
